js/adapt-articleView.js

```diff
@@ -39,6 +39,7 @@ define([
         },
 
         _blockSliderSetupEventListeners: function() {
+      this.listenTo(Adapt, "Brightcove:openTranscript", this._onBlockSliderResize);
```

Comment:
this needs to be a generic event used by both media and brightcove or it can't go in.